opentelekomcloud/provider.go

```diff
@@ -160,6 +160,7 @@ func Provider() terraform.ResourceProvider {
 			"opentelekomcloud_networking_network_v2":  dataSourceNetworkingNetworkV2(),
 			"opentelekomcloud_networking_secgroup_v2": dataSourceNetworkingSecGroupV2(),
 			"opentelekomcloud_s3_bucket_object":       dataSourceS3BucketObject(),
+			"opentelekomcloud_rds_flavors_v1":         dataSourceRdsFlavorV1(),
```

Comment:
I think you have an extra `s` in this name, should it be `opentelekomcloud_rds_flavor_v1` instead of `opentelekomcloud_rds_flavors_v1`?